src/Playwright.Tests/PageEvaluateTests.cs

```diff
@@ -633,11 +619,10 @@ private enum TestEnum
         }
 
         [PlaywrightTest(Description = "https://github.com/microsoft/playwright-dotnet/issues/1706")]
-        [Ignore("v1.22 blocker")] // actually this should not be possible what the test tries to achieve. No access to the internal data structure!
         public async Task ShouldNotReturnDisposedJsonElement()
         {
             var result = await Page.EvaluateAsync<JsonElement?>("()=> [{a:1,b:2},{a:1,b:2}]");
-            Assert.AreEqual("[{\"o\":[{\"k\":\"a\",\"v\":{\"n\":1}},{\"k\":\"b\",\"v\":{\"n\":2}}]},{\"o\":[{\"k\":\"a\",\"v\":{\"n\":1}},{\"k\":\"b\",\"v\":{\"n\":2}}]}]", result.ToString());
+            Assert.AreEqual("[{\"o\":[{\"k\":\"a\",\"v\":{\"n\":1}},{\"k\":\"b\",\"v\":{\"n\":2}}],\"id\":2},{\"o\":[{\"k\":\"a\",\"v\":{\"n\":1}},{\"k\":\"b\",\"v\":{\"n\":2}}],\"id\":3}]", result.ToString());
```

Comment:
Why do we test internal representation? Let's remove this test.

Reply:
We'll follow-up with this one in a different PR.